help/md/dnBivariatePoisson.md

```diff
@@ -11,22 +11,22 @@ Sebastian Hoehna
 ## see_also
 dnPoisson
 ## example
-	th1 ~ dnUniform(0.0,10.0)
-	th2 ~ dnUniform(0.0,10.0)
-	th0 ~ dnUniform(0.0,10.0)
-	 ~ dnBivariatePoisson(th1, th2, th0)
+    th0 ~ dnUniform(0.0, 10.0)
+	th1 ~ dnUniform(0.0, 10.0)
+	th2 ~ dnUniform(0.0, 10.0)
+	x ~ dnBivariatePoisson(th0, th1, th2)
 	x.clamp([3, 3, 3])
-	oves[1] = mvSlide(th1, delta=0.01, weight=1.0)
-	moves[2] = mvSlide(th2, delta=0.01, weight=1.0)
-	oves[3] = mvSlide(th0, delta=0.01, weight=1.0)
-	monitors[1] = mnScreen(printgen=1000,  th0)
-	ymodel = model(th1)
+	moves[1] = mvSlide(th0, delta=0.01, weight=1.0)
+	moves[2] = mvSlide(th1, delta=0.01, weight=1.0)
+	moves[3] = mvSlide(th2, delta=0.01, weight=1.0)
+	monitors[1] = mnScreen(printgen=1000, th0)
```

Comment:
`printgen=1000` floods the screen with possibly more lines than are needed for the example; suggest
```suggestion
	monitors[1] = mnScreen(printgen=20000, th0)
```

Reply:
Done